src/views/profile.hbs

```diff
@@ -1,5 +1,25 @@
 {{>header}}
+
+{{#each result}}
+
+Name: {{user.name}}
```

Comment:
name here is hidden, if you consoling the result you would see the value has stored in string ```{{'user.name'}}```

Reply:
Done Thanks ^_^